strawberryfields/backends/fockbackend/circuit.py

```diff
@@ -783,7 +783,7 @@ def measure_homodyne(self, phi, mode, select=None, **kwargs):
         )
         alpha = homodyne_sample * sqrt(m_omega_over_hbar / 2)
 
-        composed = np.dot(ops.phase(phi, self._trunc), ops.displacement(alpha, self._trunc))
+        composed = np.dot(ops.phase(phi, self._trunc), ops.displacement(np.abs(alpha), np.angle(alpha), self._trunc))
```

Comment:
(Minor suggestion): not a crucial thing at all, these could be factored out for slightly better readability
```suggestion
        r_d = np.abs(alpha)
        phi_d = np.angle(alpha)
        composed = np.dot(ops.phase(phi, self._trunc), ops.displacement(r_d, phi_d, self._trunc))
```